website/content/docs/connect/config-entries/service-splitter.mdx

```diff
@@ -1,54 +1,567 @@
+      "RequestHeaders" : {               ## map
+        "Set" : {
+          "X-Web-Version": "from-v1"
+          }
+      },
+      "ResponseHeaders" : {              ## map
+        "Set" : {
+          "X-Web-Version": "to-v1"
+          }
+      }
+    },
+    {
+      "Weight" : 10,            
+      "Service" : "service",    
+      "ServiceSubset" : "v2",          
+      "Namespace" : "target-namespace",  
+      "Partition" : "target-partition",  
+      "RequestHeaders" : {
+        "Set" : {
+          "X-Web-Version": "from-v2"
+          }
+      },
+      "ResponseHeaders" : {
+        "Set" : {
+          "X-Web-Version": "to-v2"
+        }
+      }
+    }
+  ]
+}
+```
+
+</Tab>
+
+<Tab heading="YAML" group="yaml">
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1         # string | required
+kind: ServiceSplitter                             # string | required
+metadata:                                         # object | required
+  name: config-entry-name                         # string | required
+  namespace: main                                 # string
+spec:
+  splits:                                         # list
+    - weight: 90                                  # floating point | required
+      service: service                            # string 
+      serviceSubset: v1                           # string 
+      namespace: target-namespace                 # string 
+      partition: target-partition                 # string 
+      requestHeaders:  
+        set:
+          x-web-version: from-v1                  # string 
+      responseHeaders:
+        set:
+          x-web-version: to-v1                    # string 
+    - weight: 10
+      service: service
+      serviceSubset: v2
+      namespace: target-namespace
+      partition: target-partition
+      requestHeaders:
+        set:
+          x-web-version: from-v2
+      responseHeaders:
+        set:
+          x-web-version: to-v2      
+```
+
+</Tab>
+
+</Tabs>
+
+## Specification
+
+This section provides details about the fields you can configure in the service splitter configuration entry.
+
+<Tabs>
+
+<Tab heading="HCL" group="hcl">
+
+### `Kind` 
+
+Specifies the type of configuration entry to implement. 
+
+#### Values
+
+- Default: none
+- This field is required
+- Data type: String value that must be set to `service-splitter` in HCL or `serviceSplitter` in YAML. 
+
+### `Name`
+
+Specifies a name for the configuration entry. The name is metadata that you can use to reference the configuration entry when performing operations, such as reading or writing configuration entry configuration entry settings. 
+
+#### Values
+
+- Default: Name of the node 
+- This field is required
+- Data type: String
+
+
+### `Namespace` <EnterpriseAlert inline />
+
+Specifies the [namespace](/consul/docs/enterprise/namespaces) to apply the configuration entry.
+
+#### Values
+
+- Default: None
+- Data type: String
```

Comment:
Yeah, let's add this to the topics to discuss. My preference is probably plaintext over code font because the user isn't actually writing the word "string" 🤷 , but we should still use a consistent case.